codecov.yml

```diff
@@ -15,6 +15,8 @@ codecov:
     layout: "reach, diff, flags, files"
     behavior: default
   require_ci_to_pass: true
+  ignore:
+    - "src/scenic/simulators/**"
```

Comment:
Is there a clean way to include `simulators/newtonian` and `simulators/utils` without explicitly listing all the other subfolders?

Reply:
Yes I can apply a `!` operator! I am going to make that change and let it percolate to see if it has any incompatibilities with `codecov.yml` formatting!